Mathlib/FieldTheory/IsAlgClosed/Classification.lean

```diff
@@ -73,76 +73,94 @@ end Classification
 
 section Cardinal
 
-variable {R K : Type u} [CommRing R] [Field K] [Algebra R K] [IsAlgClosed K]
-variable {ι : Type u} (v : ι → K)
+variable {R : Type u} {K : Type v} [CommRing R] [Field K] [Algebra R K] [IsAlgClosed K]
+variable {ι : Type w} (v : ι → K)
 
 theorem cardinal_le_max_transcendence_basis (hv : IsTranscendenceBasis R v) :
-    #K ≤ max (max #R #ι) ℵ₀ :=
+    Cardinal.lift.{max u w} #K ≤ max (max (Cardinal.lift.{max v w} #R)
+      (Cardinal.lift.{max u v} #ι)) ℵ₀ :=
```

Comment:
I think it's worth preserving the old statements for convenience and readability. Could you please add them as corollaries?

Reply:
done